test/__snapshots__/warningsFilter-option.test.js.snap.webpack5

```diff
@@ -0,0 +1,77 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`warningsFilter option should match snapshot for a "function" value and the "sourceMap" option is "false" (filter by message): assets 1`] = `
+Object {
+  "one.js": "",
+  "two.js": "",
```

Comment:
Empty files? Bug?

Reply:
Or webpack 5 just emit more efficient es6-code that drops by terser

Reply:
Great, I thought so